k8s/k8s.go

```diff
@@ -1825,6 +1829,20 @@ func (k *k8sOps) DeletePods(pods []v1.Pod, force bool) error {
 	return nil
 }
 
+func (k *k8sOps) DeletePod(name string, ns string, force bool) error {
```

Comment:
use this method from DeletePods in a loop instead of duplicating the code

Reply:
Fixed.